plugins/ConnectedLivingSpace/CLSPart.cs

```diff
@@ -117,36 +117,29 @@ public Part Part
             return _p.modCLS;
         }
         
-        public void Highlight(bool val)
+        public void Highlight(bool val, bool force = false)

```

Comment:
Part Highlight  has been modified to remove the delegate onmouseover function previously used to manage highlighting.  Vessel and space all trickle down to this routine as in the past, but it is also usable from third parties now.  No more "orphaned" event handlers...